h/routes.py

```diff
@@ -64,6 +64,7 @@ def includeme(config):
     config.add_route('badge', '/api/badge')
     config.add_route('token', '/api/token')
     config.add_route('api.users', '/api/users')
+    config.add_route('api.profile', '/api/profile')
```

Comment:
I think in this case it doesn't really matter. Either makes sense, so choose whichever involves less moving around.